cypress/support/config/envs.js

```diff
@@ -14,10 +14,16 @@ const config = {
     atiAnalyticsWSBucket: '598343',
   },
   local: {
-    baseUrl: 'http://localhost:7080',
-    dataUrl: 'http://localhost:7080',
-    assetUrl: 'http://localhost:7080',
-    assetOrigin: 'http://localhost:7080',
+    baseUrl: 'http://localhost.bbc.com:7080',
+    dataUrl: 'http://localhost.bbc.com:7080',
+    assetUrl: 'http://localhost.bbc.com:7080',
+    assetOrigin: 'http://localhost.bbc.com:7080',
+    assets: {
+      news: 'c0g992jmmkko',
+      newsThreeSubheadlines: 'c6v11qzyv8po',
+      persian: 'c4vlle3q337o',
+      nonExistent: 'cxvxrj8tvppo',
+    },
```

Comment:
Just out of curiosity, why were the `assets` here not needed before?

Reply:
Hmm, this isn't my code

Seems this `assets` field was in latest once upon a time, but was removed 5 days ago.  https://github.com/bbc/simorgh/commits/latest/cypress/support/config/envs.js

Looks like a mishandled merge conflict added it back here, will remove it